introduction.md

```diff
@@ -0,0 +1 @@
+Hello, I am Alex Miranda and this is my repo for the learning Github Code School Course.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.